assets/constants.ts

```diff
@@ -552,6 +552,12 @@ export = {
 
   "SKILL_DIFFICULTIES": ["Easy", "Medium", "Hard"],
 
+  "SKILL_DIFFICULTY_LABEL_TO_FLOAT": {
+    "Easy": 0.3,
```

Comment:
Do you know how we use this value in backend? If there exist another constant in backend then I would suggest using this one. (Lets not have two constant for same thing.)

Reply:
There's no mapping between the labels and the float values in feconf.py—there's only a constant for a medium skill difficulty (0.6).